impeller/entity/entity_pass.cc

```diff
@@ -124,10 +124,40 @@ std::optional<Rect> EntityPass::GetElementsCoverage(
 
       std::optional<Rect> unfiltered_coverage =
           GetSubpassCoverage(subpass, std::nullopt);
+
+      // If the current pass elements have any coverage so far and there's a
+      // backdrop filter, then incorporate the backdrop filter in the
+      // pre-filtered coverage of the subpass.
+      if (result.has_value() && subpass.backdrop_filter_proc_) {
+        std::shared_ptr<FilterContents> backdrop_filter =
+            subpass.backdrop_filter_proc_(FilterInput::Make(result.value()),
+                                          subpass.xformation_,
+                                          Entity::RenderingMode::kSubpass);
+        if (backdrop_filter) {
```

Comment:
is subpass.backdrop_filter_proc_ and no backdrop filter response an error state? Should we add a validation log for that?

Reply:
Was thinking maybe we should gracefully support nullptr results for filters that do nothing, but on second thought I think that's confusing and filters should internally elide instead (as they already do)... Added a log.